icu4c/source/common/unicode/locid.h

```diff
@@ -280,6 +280,14 @@ class U_COMMON_API Locale : public UObject {
      */
     Locale(const    Locale& other);
 
+    /**
+     * Initializes a Locale object from another Locale object.
```

Comment:
Move constructor; might leave src in bogus state.

This locale will have the same contents that the source locale had.

Reply:
Done.